api-project/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-project",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
```

Comment:
You'll want to make sure to use `--save` when installing react-router + react-router-dom (e.g. `npm install --save react-router-dom`. I went ahead and added these locally when testing the assignment.